src/main/java/com/prgrms/needit/domain/user/favorite/service/FavoriteCenterService.java

```diff
@@ -0,0 +1,77 @@
+package com.prgrms.needit.domain.user.favorite.service;
+
+import com.prgrms.needit.common.error.ErrorCode;
+import com.prgrms.needit.common.error.exception.NotFoundResourceException;
+import com.prgrms.needit.domain.user.center.dto.CenterListResponse;
+import com.prgrms.needit.domain.user.center.entity.Center;
+import com.prgrms.needit.domain.user.center.repository.CenterRepository;
+import com.prgrms.needit.domain.user.favorite.entity.FavoriteCenter;
+import com.prgrms.needit.domain.user.favorite.repository.FavoriteCenterRepository;
+import com.prgrms.needit.domain.user.member.entity.Member;
+import com.prgrms.needit.domain.user.user.service.UserService;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+public class FavoriteCenterService {
+
+	private final CenterRepository centerRepository;
+	private final FavoriteCenterRepository favoriteCenterRepository;
+	private final UserService userService;
+
+	public FavoriteCenterService(
+		CenterRepository centerRepository,
+		FavoriteCenterRepository favoriteCenterRepository,
+		UserService userService
+	) {
+		this.centerRepository = centerRepository;
+		this.favoriteCenterRepository = favoriteCenterRepository;
+		this.userService = userService;
+	}
+
+	@Transactional(readOnly = true)
+	public List<CenterListResponse> getFavCenters() {
+		Member curMember = userService.getCurMember()
+									  .orElseThrow();
+		return curMember.getFavoriteCenters()
+						.stream()
+						.map(favCenter -> new CenterListResponse(favCenter.getCenter()))
```

Comment:
취향 차이긴 하지만 map을 두 번 써서 해보는 것도 괜찮을 것 같아용!
```suggestion
						.map(FavoriteCenter::getCenter)
						.map(CenterListResponse::new)
```

Reply:
오호라 사실 저렇게 두번 이상의 로직이 들어가는 경우에 코드가 줄여지지 않아서 어쩔 수 없는 거구나 했는데....! 참고하겠습니다!!